[2025-06-12T21:40:00.504+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-06-12T21:40:00.522+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: 03_load_data_dev.load_mercadolivre manual__2025-06-12T21:39:58.556329+00:00 [queued]>
[2025-06-12T21:40:00.531+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: 03_load_data_dev.load_mercadolivre manual__2025-06-12T21:39:58.556329+00:00 [queued]>
[2025-06-12T21:40:00.532+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-06-12T21:40:00.546+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): load_mercadolivre> on 2025-06-12 21:39:58.556329+00:00
[2025-06-12T21:40:00.551+0000] {standard_task_runner.py:72} INFO - Started process 3778 to run task
[2025-06-12T21:40:00.556+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', '03_load_data_dev', 'load_mercadolivre', 'manual__2025-06-12T21:39:58.556329+00:00', '--job-id', '1094', '--raw', '--subdir', 'DAGS_FOLDER/03_load_data_dev.py', '--cfg-path', '/tmp/tmptndhoy_m']
[2025-06-12T21:40:00.558+0000] {standard_task_runner.py:105} INFO - Job 1094: Subtask load_mercadolivre
[2025-06-12T21:40:00.624+0000] {task_command.py:467} INFO - Running <TaskInstance: 03_load_data_dev.load_mercadolivre manual__2025-06-12T21:39:58.556329+00:00 [running]> on host 28e6a58e4107
[2025-06-12T21:40:00.720+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='03_load_data_dev' AIRFLOW_CTX_TASK_ID='load_mercadolivre' AIRFLOW_CTX_EXECUTION_DATE='2025-06-12T21:39:58.556329+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-12T21:39:58.556329+00:00'
[2025-06-12T21:40:00.722+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-06-12T21:40:01.470+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.9/site-packages/dlt/helpers/dbt/__init__.py:3: UserWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html. The pkg_resources package is slated for removal as early as 2025-11-30. Refrain from using this package or pin to Setuptools<81.
  import pkg_resources

[2025-06-12T21:40:01.771+0000] {config_providers_context.py:156} WARNING - Airflow variable 'dlt_secrets_toml' was not found. This Airflow variable is a recommended place to hold the content of secrets.toml.If you do not use Airflow variables to hold dlt configuration or use variables with other names you can ignore this warning.
[2025-06-12T21:40:03.839+0000] {logging_mixin.py:190} INFO - trying to load ./extraction/mercadolivre/data/raw/dev/ml_sell_data_2024-07-01T00:00:00.000-03:00_2024-10-01T00:00:00.000-03:00_0.json to sales_db
[2025-06-12T21:40:06.635+0000] {logging_mixin.py:190} INFO - Pipeline mercadolivre_data load step completed in 0.18 seconds
1 load package(s) were loaded to destination postgres and into dataset entry
The postgres destination used postgresql://***:***@pgdatabase:5432/dev_sales_db location to store data
Load package 1749764404.9995983 is LOADED and contains no failed jobs
[2025-06-12T21:40:06.637+0000] {logging_mixin.py:190} INFO - Run started at 2025-06-12 21:40:03.840808+00:00 and COMPLETED in 2.79 seconds with 4 steps.
Step extract COMPLETED in 0.80 seconds.

Load package 1749764404.9995983 is EXTRACTED and NOT YET LOADED to the destination and contains no failed jobs

Step normalize COMPLETED in 0.45 seconds.
Normalized data for the following tables:
- entry_mercadolivre: 50 row(s)
- entry_mercadolivre__payments: 52 row(s)
- entry_mercadolivre__payments__available_actions: 52 row(s)
- entry_mercadolivre__order_items: 50 row(s)
- entry_mercadolivre__tags: 118 row(s)
- entry_mercadolivre__order_items__item__variation_attributes: 44 row(s)
- entry_mercadolivre__mediations: 2 row(s)
- entry_mercadolivre__context__flows: 9 row(s)

Load package 1749764404.9995983 is NORMALIZED and NOT YET LOADED to the destination and contains no failed jobs

Step load COMPLETED in 0.42 seconds.
Pipeline mercadolivre_data load step completed in 0.18 seconds
1 load package(s) were loaded to destination postgres and into dataset entry
The postgres destination used postgresql://***:***@pgdatabase:5432/dev_sales_db location to store data
Load package 1749764404.9995983 is LOADED and contains no failed jobs

Step run COMPLETED in 2.79 seconds.
Pipeline mercadolivre_data load step completed in 0.18 seconds
1 load package(s) were loaded to destination postgres and into dataset entry
The postgres destination used postgresql://***:***@pgdatabase:5432/dev_sales_db location to store data
Load package 1749764404.9995983 is LOADED and contains no failed jobs
[2025-06-12T21:40:06.642+0000] {logging_mixin.py:190} INFO - trying to load ./extraction/mercadolivre/data/raw/dev/ml_sell_data_2024-07-01T00:00:00.000-03:00_2024-10-01T00:00:00.000-03:00_1.json to sales_db
[2025-06-12T21:40:06.987+0000] {logging_mixin.py:190} INFO - Pipeline mercadolivre_data load step completed in 0.10 seconds
1 load package(s) were loaded to destination postgres and into dataset entry
The postgres destination used postgresql://***:***@pgdatabase:5432/dev_sales_db location to store data
Load package 1749764406.7170005 is LOADED and contains no failed jobs
[2025-06-12T21:40:06.989+0000] {logging_mixin.py:190} INFO - Run started at 2025-06-12 21:40:06.644485+00:00 and COMPLETED in 0.34 seconds with 4 steps.
Step extract COMPLETED in 0.08 seconds.

Load package 1749764406.7170005 is EXTRACTED and NOT YET LOADED to the destination and contains no failed jobs

Step normalize COMPLETED in 0.09 seconds.
Normalized data for the following tables:
- entry_mercadolivre: 30 row(s)
- entry_mercadolivre__payments: 32 row(s)
- entry_mercadolivre__payments__available_actions: 32 row(s)
- entry_mercadolivre__order_items: 30 row(s)
- entry_mercadolivre__tags: 83 row(s)
- entry_mercadolivre__context__flows: 14 row(s)
- entry_mercadolivre__order_items__item__variation_attributes: 13 row(s)
- entry_mercadolivre__mediations: 2 row(s)

Load package 1749764406.7170005 is NORMALIZED and NOT YET LOADED to the destination and contains no failed jobs

Step load COMPLETED in 0.14 seconds.
Pipeline mercadolivre_data load step completed in 0.10 seconds
1 load package(s) were loaded to destination postgres and into dataset entry
The postgres destination used postgresql://***:***@pgdatabase:5432/dev_sales_db location to store data
Load package 1749764406.7170005 is LOADED and contains no failed jobs

Step run COMPLETED in 0.34 seconds.
Pipeline mercadolivre_data load step completed in 0.10 seconds
1 load package(s) were loaded to destination postgres and into dataset entry
The postgres destination used postgresql://***:***@pgdatabase:5432/dev_sales_db location to store data
Load package 1749764406.7170005 is LOADED and contains no failed jobs
[2025-06-12T21:40:06.991+0000] {logging_mixin.py:190} INFO - [WARNING] - file ./extraction/mercadolivre/data/raw/dev/temp.txt does not have .json extension and could not be loaded.
[2025-06-12T21:40:06.992+0000] {python.py:240} INFO - Done. Returned value was: -1
[2025-06-12T21:40:07.021+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-06-12T21:40:07.022+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=03_load_data_dev, task_id=load_mercadolivre, run_id=manual__2025-06-12T21:39:58.556329+00:00, execution_date=20250612T213958, start_date=20250612T214000, end_date=20250612T214007
[2025-06-12T21:40:07.070+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-06-12T21:40:07.087+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-06-12T21:40:07.090+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
